polkadot/node/core/pvf/README.md

```diff
@@ -0,0 +1,73 @@
+# PVF Host
+
+This is the PVF host, responsible for responding to requests from Candidate
+Validation and spawning worker tasks to fulfill those requests.
+
+See [the Implementer's
+Guide](https://paritytech.github.io/polkadot-sdk/book/pvf-prechecking.html#summary)
+for more information and the
+[Glossary](https://paritytech.github.io/polkadot-sdk/book/glossary.html) for an
+explanation of the terminology.
+
+## Running basic tests
+
+Running `cargo test` in the `pvf/` directory will run integration tests. Note
+that some tests run only under Linux.
+
+## Observing Logs
+
+To verify expected behavior it's often useful to observe logs. To avoid too many
+logs at once, run one test at a time:
+
+1. Add `sp_tracing::try_init_simple();` to the beginning of a test
+2. Specify `RUST_LOG=parachain::pvf=trace` before the cargo command.
+
+For example:
+
+```sh
+RUST_LOG=parachain::pvf=trace cargo test execute_can_run_serially
+```
+
+For more info on how our logs work, check [the
+docs](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/node/gum/src/lib.rs).
+
+## Running a test-network with zombienet
```

Comment:
Nit: Wouldn't this steps be better at the top of this repo ? And if we already have them can we just reference them, since they might change.